openwrt_Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2021 Aleksei Lednov
+# 
+# With best regards to the hnsd team :)
+# 
+# As always, you can do what you want with it without any warranty.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hnsd
+PKG_VERSION:=1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/handshake-org/hnsd.git
+PKG_SOURCE_VERSION:=c60b2b49adf6de92f028670f1608493dbbad1578
+PKG_MIRROR_HASH:=d2990a59bf091dda43a939ef76c5733be1fc1e727783c27b488fc49908e165ad
```

Comment:
I see `PKG_SOURCE_VERSION` is the latest commit hash -- whats `PKG_MIRROR_HASH`? Could you also add a comment about how to update these for future releases? We actually just merged an important memory leak fix and this makefile should already be updated for that.

Reply:
> whats PKG_MIRROR_HASH?

> PKG_MIRROR_HASH is also SHA256 and is used when you download the source with git protocol based on commit hash. As buildbot places the download sources to the server as .tar.xz, the next build can use the PKG_MIRROR_HASH to verify the package without calling upstream git server, and can re-use the source file instead of re-downloading it with git. (In practice you first once let the Make process to download the .tar.xz to your dl directory and then you can use that .tazr.xz to calculate the hash an place it to the Makefile.)

> Could you also add a comment about how to update these for future releases?
Sure, should I do it in the `openwrt_Makefile` also?
https://openwrt.org/docs/guide-developer/packages#use_source_repository
It's here, main part is this one:
![image](https://user-images.githubusercontent.com/51251243/134430952-ae81f8fc-3717-4a7f-891a-c455dde83707.png)

Also, there's tag variable `PKG_MAINTAINER` that can be specified in the `openwrt_Makefile` and I wanted to ask, do you want me to leave my email there? It's not a big problem for me to maintain it but I'm not available all the time unfortunately :)

Reply:
> We actually just merged an important memory leak fix and this makefile should already be updated for that.

Which version should I put in the `PKG_VERSION` then?